src/main/resources/application-azure-t.properties

```diff
@@ -46,3 +46,7 @@ xsd.cdi=${XSD_CDI}
 # Zip file constraint
 zip.entries=10
 zip.size=5000000
+
```

Comment:
I suggest deleting this file. it is no longer useful

Reply:
in the next future I would like unify `application.properties`